src/ape/contracts/base.py

```diff
@@ -126,6 +126,47 @@ def __call__(self, *args, **kwargs) -> Any:
             address=self.contract.address,
         )(*args, **kwargs)
 
+    def as_transaction(self, *args, **kwargs):
+        """
+        Convert the call to a transaction. This is useful for checking coverage
+        or checking gas costs.
+
+        Args:
+            *args: The contract method invocation arguments.
+            **kwargs: Transaction kwargs, such as value or
+              sender.
+
+        Returns:
+            :class:`~ape.api.transactions.TransactionAPI`
+        """
+        handler = ContractTransactionHandler(self.contract, self.abis)
+        return handler.as_transaction(*args, **kwargs)
```

Comment:
Why not using the property?
```suggestion
        return self.transact.as_transaction(*args, **kwargs)
```

Reply:
It reads funny but ya that works. `transact` is technically a property but it is a handler object so you can call it